client/js/components/month/month.js

```diff
@@ -1,12 +1,34 @@
 /** @jsx React.DOM */
 var React = require('react');
-var MonthCalendarContainer = require('./month-calendarcontainer.js');
+var MonthEventsContainer = require('./month-eventcontainer.js');
+var MonthStore = require('../../stores/month-store.js');
+var MonthHeader = require('./month-header.js');
+var MonthData = require('../../monthData.js');
+
+function monthData() {
+  return {
+    gigs: MonthStore.getMonthData()
+  };
+}
 
 var Month = React.createClass({
-  render: function () {
+  getInitialState: function() {
```

Comment:
Super nit picky, but JSLint is going to throw an error anytime we don't have proper spacing on function definitions. Should be function () {} where everything is separated by spaces. If you write a function expression in sublime, you'll notice the syntax highlighting doesn't even work properly if one of the spaces are missing.

Could you modify this throughout your pull request? I'm trying to do it retroactively as I make other updates to individual files.